app/src/main/java/gov/hhs/cdc/trustedintermediary/external/reportstream/ReportStreamLabOrderSender.java

```diff
@@ -85,7 +87,11 @@ protected String requestToken() {
         try {
             senderToken =
                     jwt.generateSenderToken(
-                            CLIENT_NAME, RS_DOMAIN_NAME, "pemKey", CLIENT_NAME, 300);
+                            CLIENT_NAME,
+                            RS_DOMAIN_NAME,
+                            secrets.getKey("AZURE_KEY_NAME"),
```

Comment:
Should I use a container for this so we check what we got from Azure?

Reply:
You mean test in a Docker container to see if we actually retrieve the secret?  Or...?

Reply:
meant for the Azure env.